modules/auxiliary/mitm.py

```diff
@@ -57,22 +66,31 @@ def start(self):
 
         PORT_LOCK.release()
 
-        args = [
-            mitmdump, "-q",
-            "-s", "\"%s\" %s" % (script, self.task.options.get("mitm", "")),
-            "-p", "%d" % self.port,
-            "-w", outpath
-        ]
-
+        # Assemble the command to execute
+        command = '%s -q -s "%s" %s -p %d -w %s' % (
+            mitmdump,
+            os.path.join(CUCKOO_ROOT, script),
+            self.task.options.get("mitm", ''),
+            self.port,
+            outpath,
+        )
         mitmlog = os.path.join(CUCKOO_ROOT, "storage", "analyses",
                                "%d" % self.task.id, "mitm.log")
 
         mitmerr = os.path.join(CUCKOO_ROOT, "storage", "analyses",
                                "%d" % self.task.id, "mitm.err")
 
-        self.proc = subprocess.Popen(args,
+        # Store the TLS master secrets for TLS decryption with Wireshark
+        tls_secrets_path = os.path.join(CUCKOO_ROOT, "storage", "analyses",
+                                 "%d" % self.task.id, "mitm_tls_secrets.log")
+
+        self.proc = subprocess.Popen(shlex.split(command),
```

Comment:
arguments passed into Popen should probably be a list and not a split string in cases like this where you can control the arguments.